### ISSUE TITLE
different preprocess method for images and labels

### ISSUE THREAD

**Issue by fido20160817** [yolu1055/conditional-glow#5]
Hi, in datasets.py:
```python
def preprocess(x, scale, bias, bins, noise=False):

    x = x / scale
    x = x - bias

    if noise == True:
        if bins == 2:
            x = x + torch.zeros_like(x).uniform_(-0.5, 0.5)
        else:
            x = x + torch.zeros_like(x).uniform_(0, 1/bins)
    return x
```
I notice that you set 'scale' and 'bias' for rgb and label respectively. Why you define different values for the two kinds of data or what's the reason for this?
